.github/workflows/build.yaml

```diff
@@ -35,3 +38,28 @@ jobs:
           tags: "ghcr.io/${{ github.repository }}:${{ github.sha }}"
           cache-from: type=gha,scope=${{ github.repository }}
           cache-to: type=gha,scope=${{ github.repository }},mode=max
+
+  trivy-scan:
+    needs: build
+    name: Trivy Vulnerability Scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Aqua Security Trivy Scan
+        uses: aquasecurity/trivy-action@0.9.1
+        id: trivy_scan
+        with:
+          image-ref: ghcr.io/${{ github.repository }}:${{ github.sha }}, redis:alpine
+          format: table # more readable for humans
```

Comment:
I don't think codeql will accept this format, it has to be sarif.

Reply:
Oh ok, i will change this

Reply:
I changed to json, normally it will work now